terraform-datadog-sqs/main.tf

```diff
@@ -1,6 +1,9 @@
+locals {
+  identifierLabel = var.identifier ? "(${var.identifier})" : ""
+}
 
 resource "datadog_monitor" "sqs_number_of_messages_monitor" {
-  name = trimspace("${var.service_name_readable} – SQS - Number of messages %{if var.identifier != ""}(${var.identifier})%{else}%{endif}")
```

Comment:
Parser checkovu si s tím nějak nedokázal poradit, tak jsem to jen přepsal.

<img width="768" alt="CleanShot 2023-07-14 at 14 12 05@2x" src="https://github.com/FigurePOS/terraform-modules/assets/215660/fe7b2319-679e-45e9-b9cf-ed38d2f00379">